Lib/test/support/__init__.py

```diff
@@ -1168,6 +1168,16 @@ def requires_limited_api(test):
         return unittest.skip('needs _testcapi and _testlimitedcapi modules')(test)
     return test
 
+
+TEST_MODULES = sysconfig.get_config_var('TEST_MODULES')
```

Comment:
Perhaps name this `TEST_MODULES_ENABLED` and make it a boolean.

Reply:
That sounds like a good idea.
Also, IMO if `TEST_MODULES` has an unknown value the tests should *not* be skipped -- they should fail.

Reply:
> Also, IMO if `TEST_MODULES` has an unknown value the tests should _not_ be skipped -- they should fail.

Well, you might as well make the whole test suite fail in that case; if `TEST_MODULES` has an unknown value, something is more than a little bit off.